front/src/config/i18n/en.json

```diff
@@ -253,6 +253,51 @@
       "apiKeyPlaceholder": "Enter DarkSky API key",
       "saveButton": "Save",
       "instructionsToUse": "To use this integration, go to the dashboard and create a weather box."
+    },
+    "mqtt": {
+      "title": "MQTT",
+      "deviceTab": "Devices",
+      "setupTab": "Setup",
+      "device": {
+        "title": "MQTT Devices",
+        "search": "Search devices",
+        "noDevices": "No MQTT devices added yet.",
+        "noNameLabel": "No name",
+        "nameLabel": "Name",
+        "roomLabel": "Room",
+        "featuresLabel": "Features",
+        "noFeatures": "No features",
+        "saveButton": "Save",
+        "deleteButton": "Delete",
+        "returnButton": "Return back",
+        "editButton": "Edit details",
+        "notFound": "Requested device not found.",
+        "backToList": "Back to device list",
+        "saveError": "Error saving or deleting device"
+      },
+      "feature": {
+        "nameLabel": "Name",
+        "namePlaceholder": "Enter feature name",
+        "externalIdLabel": "Feature external ID",
+        "externalIdPlaceholder": "Feature MQTT message key",
+        "minLabel": "Minimum value",
+        "minPlaceholder": "Enter feature minimum value",
+        "maxLabel": "Maximum value",
+        "maxPlaceholder": "Enter feature maximum value",
+        "addButton": "Add feature",
+        "deleteLabel": "Delete feature"
+      },
+      "setup": {
+        "title": "MQTT configuration",
+        "urlLabel": "Broker URL",
+        "urlPlaceholder": "Ex: http://[mqtt-broker-address]:[port]",
```

Comment:
It's not really how a MQTT address looks like, no? More like mqtt:// ?

Reply:
You're right :)